cgi-bin/LJ/Talk.pm

```diff
@@ -2256,6 +2256,40 @@ sub init_s2journal_js {
         ) ) if $opts{lastn};
 }
 
+# convenience function for keyboard shortcuts
+# args: $remote and $p for adding javascript to header
+# returns: nothing, just calls need_res
+sub init_s2journal_shortcut_js {
+    my ( $remote, $p ) = @_;
+
+    if ( $remote ) {
+        my $connect_string = "";
+        if ( $remote->prop( "opt_shortcuts" ) ||  $remote->prop( "opt_shortcuts") ) {
```

Comment:
Did you mean to look at a different property here? This seems like the same one twice.

nit: Also, same comment about deep nesting: you could just return if these are false.